helical/models/classification/neural_network.py

```diff
@@ -32,18 +39,20 @@ def compile(self, num_classes: int, input_shape: int) -> None:
         """
 
         self.num_classes = num_classes
-        self.input_shape = (input_shape,)
-
-        self.model = Sequential()
-        self.model.add(Dense(256, activation='relu', input_shape=self.input_shape))
-        self.model.add(Dropout(0.4))
-        self.model.add(Dense(64, activation='relu'))
-        self.model.add(Dropout(0.4))
-        self.model.add(Dense(self.num_classes, activation='softmax'))
-
-        self.optimizer = Adam(learning_rate=self.learning_rate)
-        self.f1_metric = F1Score(average='macro')
-        self.model.compile(loss=self.loss, optimizer=self.optimizer, metrics=[self.f1_metric])
+        self.input_shape = input_shape
+        self.model = nn.Sequential(
+            nn.Linear(input_shape, 256),
+            nn.ReLU(),
+            nn.Dropout(0.4),
+            nn.Linear(256, 64),
+            nn.ReLU(),
+            nn.Dropout(0.4),
+            nn.Linear(64, num_classes)
+        )
```

Comment:
I know this is how we had it before, but do we want this to be configurable at all?

Reply:
there is a TODO but so far there has been no request to make it configurable